src/components/flashMessage.js

```diff
@@ -0,0 +1,34 @@
+import React, { useState, useEffect } from 'react';
+import Bus from '../lib/bus';
+
+const FlashMessage = () => {
+  const intialMessage = { content: '', type: '' };
+  const [message, setMessage] = useState(intialMessage);
+
+  useEffect(() => {
+    Bus.addListener('flash', ({ content, type }) => {
+      displayMessage(content, type);
+    });
```

Comment:
I think we can add this listener in another `useEffect` that has an empty array as a dependency so that we only `addListener` and `removeAllListeners` once. :)

Reply:
yeah makes sense
Thank youu 👍